docs/references/protocol/transactions/types/ledgerstatefix.md

```diff
@@ -0,0 +1,68 @@
+---
+seo:
+    description: Repair corruptions to the XRP ledger.
+labels:
+  - Utilities, troubleshooting
+---
+# LedgerStateFix
+[[Source]](https://github.com/XRPLF/rippled/blob/master/src/xrpld/app/tx/detail/LedgerStateFix.cpp "Source")
+
+
+`LedgerStateFix` is a general purpose transaction used to fix specific issues affecting the XRP ledger. You submit the transaction with the `LedgerFixType` value set to indicate the particular  error state to correct.
```

Comment:
Isn't this transaction gated by an amendment? Which one?

Reply:
I couldn't find it, could you help track it down?